src/global/interfaces/table-header.ts

```diff
@@ -0,0 +1,7 @@
+export interface TableHeader<T> {
+  key: keyof T | 'actions';
+  title: string;
+  align: 'start' | 'center' | 'end';
+  width?: number;
+  value?: (item: Record<string, any>, fallback?: any) => any;
```

Comment:
Na pewno rekord będzie zawierał any? fallback to nie będzie po prostu funkcja z voidem? to samo typ zwracany całej funkcji value, moze też void?